emgapi/migrations/0017_auto_20240129_1401.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 3.2.18 on 2024-01-29 14:01
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('emgapi', '0016_auto_20240117_1757'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='analysisjob',
+            name='last_indexed',
+        ),
+        migrations.RemoveField(
+            model_name='study',
+            name='last_indexed',
+        ),
+        migrations.AddField(
+            model_name='analysisjob',
+            name='last_ebi_search_indexed',
+            field=models.DateTimeField(blank=True, db_column='LAST_EBI_SEARCH_INDEXED', help_text='Date at which this model was last included in an EBI Search initial/incremental index.', null=True),
+        ),
```

Comment:
Can we have the migration to a `RenameField` operation instead please, on the Study and AnalysisJob `last_indexed` fields? That way, we won't lose track of the existing EBI Indexation state.

Reply:
Yep, the PR is "outdated" now.